openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -139,10 +139,10 @@ <h3 class="message__title">{% trans "We could not submit your response" %}</h3>
 
                 <ul class="list list--actions">
                     <li class="list--actions__item">
-                        <a aria-role="button" href="#" id="step--response__submit"
-                            class="action action--submit step--response__submit {{ submit_enabled|yesno:",is--disabled" }}">
+                        <button type="submit" id="step--response__submit"
```

Comment:
Excellent.  Thanks for doing that.  adding type=submit is optional as that is the implied value.  If you don't declare a type, it is treated like a submit.   This is generally what you want, and even if that is not what you want (submit buttons are used with forms) it usually doesn't cause any issues if the button is just a trigger for a javascript event and is not related to  a form.  It's up to you if you want to explicitly declare this.